_posts/2016-10-05-How-to-extract-ZIP-or-RAR-archives-on-your-iOS-device-for-free.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: How to extract ZIP or RAR archives on your iOS device for free
+twitter_username: pietbrauer
+categories: blog
+readtime: 3 min read
+header_image: "/img/2016-10-05-Extract-Archives-Header.jpg"
+---
+
+__The internet is full of technology that saves device storage space and bandwidth to get files to your device faster. But iOS lacks a built-in way to extract ZIP or RAR files leaving you staring at an archive instead of enjoying the latest home movie of a friend. But we got you covered, 100% for free, without ads and iPad Pro optimized.__
+
+
+Let me introduce [Documents 5](https://itunes.apple.com/us/app/documents-5-file-manager-pdf/id364901807?mt=8) to you. [Documents 5](https://itunes.apple.com/us/app/documents-5-file-manager-pdf/id364901807?mt=8) is an app developed by [Readdle](https://readdle.com), a company mostly know for [PDF Expert](https://readdle.com/products/pdfexpert5). It is a file manager unlike any other, which understands many file formats from JPEG and PNG to DOCX and Numbers. It connects to your FTP, WebDAV, Dropbox, Box and even Office 365 account.
+It truly is a one app fits all file manager. You can transfer files between storage providers, import them into other apps, organize everything in folders and add favorites. It can even play mp3 files in a playlist, a feature Apple still has not added to the stock music app, and play many video formats.
+
+But back to our initial problem of the archive someone sends to you. For this test I downloaded the [Ulysses](https://itunes.apple.com/us/app/ulysses/id950335311?mt=8), the app I do all my writing in, Press Kit from their website. It is a 206 MB ZIP archive full of beautiful images of people writing books in different locations. To download bigger files, I like to use [iCab Mobile](https://itunes.apple.com/en/app/icab-mobile-web-browser/id308111628?mt=8) which excels when it comes to downloading files (This also works with files downloaded in Safari).
```

Comment:
'...Press Kit from their website.' 
Does this make sense?
Feels like there should be '.' after '...writing in.'